terminal_bsd.go

```diff
@@ -1,4 +1,5 @@
 // +build darwin freebsd openbsd netbsd dragonfly
+// +build !appengine
```

Comment:
It means, `(darwin OR freebsd OR openbsd OR netbsd OR dragonfly) AND !appengine`.